### PR TITLE
Change incorrectly typed enums to bitflags in audio_format module. Increment version to 0.3 for breaking changes.

### PR DESCRIPTION
**Pull request by mitchmindtree**
@yupferris does this look good to you? I'm about to go for coffee, but were there any other types that needed bitflag-ification? If so, I can add them to this PR. Otherwise if there are no issues, feel free to merge and publish this.

### CODE REVIEW

File: src/audio_unit/audio_format.rs [hunk 1]
```diff
@@ -6,61 +6,191 @@
...
+    /// MPEG-1/2, Layer 3 audio.
+    ///
+    /// **Available** in OS X v10.3 and later.
     MPEGLayer3,                           // = 778924083,
-    TimeCode(Option<AudioTimeStampFlag>), // = 1953066341,
+    /// A stream of IOAudioTimeStamp structures.
+    ///
+    /// **Available** in OS X v10.2 and later.
+    TimeCode(Option<AudioTimeStampFlags>), // = 1953066341,
+    /// A stream of MIDIPAcketList structures where the time stamps in the MIDIPacket structures
```
Comment (yupferris) [inline src/audio_unit/audio_format.rs]: I have no idea HOW I spotted this, but there's a slight capitalization error; should be `MIDIPacketList`, not `MIDIPAcketList`

File: src/audio_unit/audio_format.rs [hunk 2]
```diff
@@ -137,132 +267,237 @@ impl AudioFormat {
...
-            7           => Some(LinearPCMFlag::FlagsSampleFractionShift),
-            8064        => Some(LinearPCMFlag::FlagsSampleFractionMask),
-            _           => None,
+/// A wrapper around the const **StandardFlags**.
+pub mod standard_flags {
+    bitflags! {
+        /// Standard flags for use in the **F60958AC3** **AudioFormat** variant.
+        ///
+        /// Note: In the original Core Audio API these are consolidated with what we have named the
+        /// **StandardFlags** and **AppleLosslessFlag** types under the `AudioFormatFlag` type. We have
```
Comment (yupferris) [inline src/audio_unit/audio_format.rs]: should be `**AppleLosslessFlags**` (missing "s")

File: src/audio_unit/audio_format.rs [hunk 3]
```diff
@@ -137,132 +267,237 @@ impl AudioFormat {
...
+            /// The linear PCM flags contain a 6-bit bitfield indicating that an integer format is to
+            /// be interpreted as fixed point.
+            ///
+            /// The value indicates the number of bits are used to represent the fractional portion of
+            /// each sample value.
+            ///
+            /// This constant indicates the bit position (counting from the right) of the bitfield in
+            /// `mFormatFlags` field.
+            ///
+            /// TODO: Review whether or not this flag indicates that we need to treat LinearPCM format
```
Comment (yupferris) [inline src/audio_unit/audio_format.rs]: I don't know how to handle this yet; TODO is fine for now probably :)

**Comment by yupferris** [comment]
I'm done going over these changes; great stuff! After the notes I added are addressed I'm happy with this being merged :+1:

**Comment by mitchmindtree** [comment]
Haha amazing hawk-eye typo review :+1:

Just committed the fixes, will merge and publish!

**Comment by mitchmindtree** [comment]
Published as **0.3.0**.

**Pull request merged**